src/graphql/helpers.ts

```diff
@@ -375,3 +383,35 @@ export function formatSubscription(subscription) {
   });
   return subscription;
 }
+
+async function getControllerDomains(address: string): Promise<string[]> {
+  type JsonRpcResponse = {
+    result: string[];
+    error?: {
+      code: number;
+      message: string;
+      data: any;
+    };
+  };
+
+  try {
+    const response = await fetch(process.env.STAMP_URL, {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        method: 'lookup_domains',
```

Comment:
But we have other conditions to get space controller 🤔  Refer to this function https://github.com/snapshot-labs/snapshot.js/blob/master/src/utils.ts#L590-L607

Reply:
This is true, but this is not the method used by the UI to fetch all spaces controlled by an user.

See https://github.com/snapshot-labs/snapshot/blob/514f12c84d5986ef86f87193057d06d9b5c73353/src/views/ProfileAbout.vue#L59

The DNS check seems to be very hard to check live